### ISSUE TITLE
ML: VGG16 Model #1 (24 Layers): Configuration#1: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#265]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs

Epoch 00030: val_accuracy improved from 0.82210 to 0.82210, saving model to /wsu/home/gy/gy40/gy4065/hm.jetscapeml.data/simulation_results/simulation-results-deep-model-cnn-01-1200K-config-01/Models_PP19_vs_LBT_Hadron_ch1/best_model.h5
840000/840000 [==============================] - 309s 368us/sample - loss: 0.8033 - accuracy: 0.8092 - val_loss: 0.7783 - val_accuracy: 0.8221

**Comment by hmehryar** [comment]
epoch 50 

Epoch 50/50
839936/840000 [============================>.] - ETA: 0s - loss: 0.6099 - accuracy: 0.8146  
Epoch 00050: val_accuracy did not improve from 0.82648
840000/840000 [==============================] - 323s 385us/sample - loss: 0.6099 - accuracy: 0.8146 - val_loss: 0.5948 - val_accuracy: 0.8264
